README.md

```diff
@@ -1 +1,60 @@
 # Core Domain Charts
+
+Core Domain Charts help you to visualise the strategic importance of each sub-domain or business capability in your software architecture allowing you to make business model-aligned architectural decisions.
+
+[Core Domains]() are the parts of your domain where the expected ROI curve is greatest, and deserve the highest focus.
+
+The true power of this technique is the conversations that it triggers, especially cross-discipline. Complexity is something that engineers can gauge whereas business differentiation is provided by product managers or business stakeholders.
+
+![alt text](resources/core-domain-chart-template.jpg "Core Domain Chart")
+
+## How to Use
+There are a variety of ways the Core Domain Chart can be used, but it's important not to try to and visualise all of the possible information in a single diagram. Below are three flavours to choose from.
+
+### Sub-domain/Bounded Context Portfolio
+This is the simplest flavour. Simply plot each of your sub-domains or bounded contexts on the chart to get a relative sense of ordering between them.
+
+### Context Map With Team Topologies
+You can augment your Core Domain Charts with the dependencies between your bounded contexts and the type of [Team Topologies](https://github.com/TeamTopologies) Interaction Mode in play.
+
+### Architecture Migration
+With a slight tweak of the y axis label, core domain charts can be used to plan the order in which you migrate from your current architecture to your target architecture.
+
+![alt text](resources/architecture-migration-core-domain-chart.jpg "Architecture Migration Core Domain Chart")
```

Comment:
maybe add an example of all three flavours?

Reply:
Agreed, people like to see and learn by example.

Reply:
Thanks. Examples definitely needed. I'm going to add those in a future PR.